src/progress_bar_exercise/ProgressBar/progress-bar-no-break.component.js

```diff
@@ -0,0 +1,48 @@
+import { useCss } from "kremling";
+import React, { useEffect, useRef, useState } from "react";
+
+export default function ProgressBarNoBreak({ loading }) {
+  const [opacity, setOpacity] = useState(1)
+  const [width, setWidth] = useState('0%')
+  const [transition, setTransition] = useState('15s ease, opacity 1s ease-in-out')
```

Comment:
Here a significant difference is keeping transition in state. This permits us to slowly animate to 90% but then rush to 100% if desired.